packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -111,6 +111,11 @@
 	.dataviews-pagination {
 		z-index: z-index(".edit-site-patterns__dataviews-list-pagination");
 	}
+
+	.is-badge:has(.edit-site-patterns__field-sync-status-fully) {
```

Comment:
Wouldn't this work?

```suggestion
	.edit-site-patterns__field-sync-status-fully {
```

Reply:
That suggestion would produce this: 

<img width="539" alt="Screenshot 2024-04-04 at 13 55 05" src="https://github.com/WordPress/gutenberg/assets/846565/620d1fd6-c692-4d9f-914a-70dcd364d992">

The styles need to be applied to the element that contains `.edit-site-patterns__field-sync-status-fully`. 

We can probably do that without the `is-badge` class though. I'll try something.